atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/memory/InMemoryAtlasDbFactory.java

```diff
@@ -83,15 +87,22 @@ public TimestampService createTimestampService(KeyValueService rawKvs) {
     }
 
     public static SerializableTransactionManager createInMemoryTransactionManager(AtlasSchema schema) {
+        return createInMemoryTransactionManager(ImmutableSet.of(schema));
+    }
+
+    public static SerializableTransactionManager createInMemoryTransactionManager(Set<AtlasSchema> schemas) {
```

Comment:
can we have a varargs

```
createInMemoryTransactionManager(AtlasSchema schema, AtlasSchema... otherSchemas)
```

?

feels like one fewer boilerplate we can apply.